### DISCUSSION TITLE
V2?

### DISCUSSION THREAD

**Discussion by ricardo-re-azevedo** [nealcarney/ESPHome-MaxxFan-controller#2] [General]
Looking to do something similar for my maxxfan. Any word on an updated version of the board?

**Comment by nealcarney** [comment]
Hi Ricardo.  If you follow the link to the EasyEDA designs files (https://oshwlab.com/ncarney/maxxfan-controller) you will see that the last version is 1.2.  It is unlikely I will update the project further.  I created version 1.2 based on what I discovered was wrong with version 1.0, but I never had the v 1.2 board produced.  I'm pretty sure it should work.  I used the 1.0 but then had to perform surgery on it to correct the things that were wrong or left off.  The main thing wrong with 1.0 was there was an extra relay for open-close which was definitely an error because it was wired in parallel with the other relays.  Rather than use a separate relay for open-close you need to simultaneously close the other two relays, which you will see in the ESPHome code which has been working fine.  It looks like I also remove the relay for auto mode in 1.2 because I ended up not using it (I can monitor temperature and control fan via ESPHome).  Another change from v1.0 to v1.2 is that I ended up using two IR transmitters instead of one (one dedicated to the fann and one dedicated to the AC unit).  You may just need one if you are only using for MaxxFan.  I think in v1.2 I also added a Schottky diode after the buck converter.  This is in case you incorrectly power the board simultaneously with power from 12V source and also power from USB port on ESP32.  I have not added the diode in my modified 1.0 board. But I did fry a buck converter by powering both.  Hope the info above helps.  Good luck with your project.
